packages/cozy-harvest-lib/src/components/TwoFAForm/index.jsx

```diff
@@ -89,6 +90,7 @@ export class TwoFAForm extends PureComponent {
               busy={submitting}
               disabled={submitting || !twoFACode}
               extension="full"
+              onClick={this.handleSubmit}
```

Comment:
Did you tried to add a `type=submit`?

Reply:
The CozyUI button is `type=submit` by default

Reply:
This is a bit surprising !

Reply:
Actually `submit` is the default type for raw `<button>`s. See https://codesandbox.io/s/frosty-leaf-dkfb9